include/deal.II/base/std_cxx11/bind.h

```diff
@@ -24,10 +24,13 @@
 #  include <functional>
 
 DEAL_II_NAMESPACE_OPEN
-// in boost, the placeholders _1, _2, ... are in the global namespace. in
+// In boost, the placeholders _1, _2, ... are in the global namespace. In
 // C++0x, they are in namespace std::placeholders, which makes them awkward to
-// use. import them into the deal.II::std_cxx11 namespace instead and do them
-// same below if we use boost instead.
+// use. Import them into the dealii::std_cxx11 namespace instead and do them
+// same below if we use boost instead. Namespace placeholders is also defined
```

Comment:
say "Namespace 'placeholders'" to make clear that you're talking about a proper word here.